static/js/redux/ui/semesterly.tsx

```diff
@@ -223,11 +223,9 @@ const Semesterly = () => {
                 <a
                   target="_blank"
                   rel="noopener noreferrer"
-                  href="https://discord.gg/txYbphsAV7"
-                  // TODO: add discord logo correctly
+                  href="https://discord.gg/HmnwdbehBh"
                 >
-                  <i className="fab fa-discord" />
```

Comment:
Did you mean to intend the Discord logo?

Reply:
Ah the tag never worked, I forgot. Leave the TODO though

Reply:
I'll make it an issue

Reply:
Okay